### ISSUE TITLE
[REQ] | Compatibility with 1.8

### ISSUE THREAD

**Issue by TrademarkTM** [bensku/Skript#159]
Hey!
This is a great fork, but there's no compatibility for 1.8. I'm using ViaVersion (supports 1.8 - 1.10), but I don't want to update to 1.9 or 1.10 because the PvP is completely broken (my player's wouldn't like it).

Please, add support for 1.8, it cannot be that hard, and I don't want to use other versions because the alphabetical order and convert to upper/lowercase are extremely useful to me, and this is the most up-to-date version.

Thank you!

**Comment by TheBentoBox** [comment]
Just upgrade and use a PvP reversion plugin [like this one](https://www.spigotmc.org/resources/1-8-pvp-for-1-9-and-1-10.19291/). I'm not Bensku's dad or anything so I can't tell him what action to take, but just personally speaking I think there are issues in this fork that are more important to address than adding compatibility for an outdated version, especially when plugins like that exist.

**Comment by JKGamerxD** [comment]
you're completely wrong, add support for an outdated version like 1.8 will be really hard and if you need support for 1.8, there is the Mirreski fixes or other versions.

**Comment by bensku** [comment]
I just fixed one thing that crashed under Minecraft 1.8. Managed to run Spigot 1.8.8 without Skript completely crashing.

1.8 will never be officially supported by me, though. I might fix something occasionally, but it is only if I
1) Feel that the fix will help also 1.9+ users or
2) Have extra time for some reason (doesn't happen often) or
3) Just want to fix it for coding challenge (since it is not easy usually)

I recommend you try with next Skript update and report if anything works :)
